labcodes/confetti-ds
# [CSYS-364] Migrating Abstract Dropdown and TagDropdown

Link to task:
[CSYS-362](https://labcodes.atlassian.net/browse/CSYS-364)

Staging app: https://CSYS-364-migrate-abstract-dropdown--confetti-storybook.netlify.app/

How to test:
After running your project locally, run the AbstractDropdown.test.js and TagDropdown.test.jsto check if the TagDropdown component is working as expected.

Description of your solution:

Removing previous mentions from Class Components and moving the Props outside the functiomemoized getSelectedOptionIndex callback function. We also updated the AbstractDropdown's test to handle the new trigger processes for click up and down events.

Screenshots (when applicable): NA
